proposals/partial-properties.md

```diff
@@ -107,6 +108,15 @@ partial class C3
     // Error: implementation of 'Prop' cannot have a 'set' accessor because the definition does not have a 'set' accessor.
     public partial string Prop { get => field; set => field = value; }
 }
+
+partial class C4
+{
+    public partial string this[string s = "a"] { get; set; }
+    public partial string this[string s] { get => s; set { } } // ok
+
+    public partial string this[int i, string s = "a"] { get; set; }
+    public partial string this[int i, string s = "a"] { get => s; set { } } // CS1066: The default value specified for parameter 'x' will have no effect because it applies to a member that is used in contexts that do not allow optional arguments
```

Comment:
```suggestion
    public partial string this[int i, string s = "a"] { get => s; set { } } // CS1066: The default value specified for parameter 's' will have no effect because it applies to a member that is used in contexts that do not allow optional arguments
```